src/app/contexts/EventTrackingContext/index.test.jsx

```diff
@@ -136,11 +136,6 @@ describe('Error handling', () => {
 
     expect(trackingData).toEqual({});
     expect(errorMessage).toBeUndefined();
-    expect(console.error).toHaveBeenCalledWith(
```

Comment:
why was this removed?

Reply:
When we used the getContentId function, we attempted to call split on curie: https://github.com/bbc/simorgh/blob/latest/src/app/lib/analyticsUtils/index.js#L232

In this test, we don't provide curie and an error is thrown, caught here: https://github.com/bbc/simorgh/blob/latest/src/app/containers/ATIAnalytics/params/index.js#L123 

I've removed the use of split and replaced it with path, so we no longer see the error expected in this assertion.